src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,7 @@
+$white: #fff;
+$lightGray: #f3f3f3;
+$darkGray: #616070;
+$blue: #00acdc;
+$darkBlue: #060b35;
+$baseFontSize: 12px;
+$baseFontWeight: 400;
```

Comment:
This is default value, so you can just remove it everywhere
```suggestion
```

Reply:
+++